### DISCUSSION TITLE
是否可以破解网页端的投稿工具？以支持多P上传？

### DISCUSSION THREAD

**Discussion by SirnoChan** [the1812/Bilibili-Evolved#2756] [功能建议 / Ideas]
目前B站的网页端不能分P，只能在APP、必剪和投稿工具中投稿多P，是否有插件作者愿意加入对投稿功能的破解呢？[记一次B站投稿工具逆向](https://fortunedayssss.github.io/2020/05/20/%E8%AE%B0%E4%B8%80%E6%AC%A1B%E7%AB%99%E6%8A%95%E7%A8%BF%E5%B7%A5%E5%85%B7%E9%80%86%E5%90%91.html)

**Comment by the1812** [comment]
投稿工具我不熟, 有谁感兴趣可以研究下(
